src/common/exif.cc

```diff
@@ -1282,6 +1282,26 @@ static bool _exif_decode_exif_data(dt_image_t *img, Exiv2::ExifData &exifData)
         img->exif_crop = 0.0f; // Will be shown as "no data" in the image information module
     }
 
+    // For a pair of cameras with known sensor sizes we can't calculate the crop factor
```

Comment:
Can we introduce here a table (struct) with camera models and the corresponding crop factor. This will make it easier to extend in the future.

Reply:
Also we test just for model - should we entertain a possibility there could be different makers with the same model? In theory it should never happen because of trademark violation...

Reply:
Yes, it will definitely be better. Done.

Reply:
> Also we test just for model - should we entertain a possibility there could be different makers with the same model? In theory it should never happen because of trademark violation...

Yes, it should never happen. If, very theoretically, such a situation arises, only then should we simply add the model to an alternative structure where there is also a maker. And do a separate loop for the structure with the maker, since there is no point in checking the maker for those cases where model matching is enough.